drivers/modem/gsm_ppp.c

```diff
@@ -24,6 +25,7 @@ LOG_MODULE_REGISTER(modem_gsm, CONFIG_MODEM_LOG_LEVEL);
 
 #include <stdio.h>
 
+#define GSM_UART_DEV_ID                 DT_INST_BUS(0)
```

Comment:
Maybe `GSM_UART_NODE` instead of `GSM_UART_DEV_ID`?

Reply:
Done